### PR TITLE
Fix broken links to images

### PR DESCRIPTION
**Pull request by snizhana-dynnyk**
changed image locations consistently to reside in the same folder where the markdown resides they are referenced in and updated links:

- content/basics/aws-architecture
- content/basics/azure-architecture
- content/basics/on-prem-architecture

For review:
https://github.com/giantswarm/docs/blob/snizhana-dynnyk-patch/src/content/basics/onprem-architecture/index.md
https://github.com/giantswarm/docs/blob/snizhana-dynnyk-patch/src/content/basics/aws-architecture/index.md
https://github.com/giantswarm/docs/blob/snizhana-dynnyk-patch/src/content/basics/azure-architecture/index.md

### CODE REVIEW

File: src/content/basics/aws-architecture/index.md [hunk 1]
```diff
@@ -13,33 +13,33 @@ Giant Swarm's Architecture is split into two logical parts, one being the Contro
...
 
 Access to those interfaces can be configured to work with the customers' Identity Management System.
 
 ## Giant Swarm tenant cluster
 
-![Tenant Cluster Architecture](/img/architecture-aws-tenant-cluster.png)
+![Tenant Cluster Architecture](architecture-aws-tenant-cluster.png)
 
-Via the Giant Swarm API, our [CLI](https://github.com/giantswarm/gsctl), or our Happa interface, you can start Tenant Clusters of different sizes. There's a selection of recommended EC2 instance types, which can be adjusted if needed. 
+Via the Giant Swarm API, our [CLI](https://github.com/giantswarm/gsctl), or our Happa interface, you can start Tenant Clusters of different sizes. There's a selection of recommended EC2 instance types, which can be adjusted if needed.
```
Comment (Oshratn) [inline src/content/basics/aws-architecture/index.md]: I'd like us to start eliminating Happa from documentation and use "our web UI" instead.

File: src/content/basics/azure-architecture/index.md [hunk 2]
```diff
@@ -14,29 +14,29 @@ Giant Swarm's Architecture is split into two logical parts, one being the Contro
 
 ## Giant Swarm control plane
 
-![Control Plane Architecture](/img/architecture-azure-control-plane.png)
+![Control Plane Architecture](architecture-azure-control-plane.png)
 
 The above diagram shows our control plane setup running with two bastion hosts in different availability zones. All instances are in a private subnet and only accessible through the bastion hosts. Bastions hosts are available via Azure VPN connection only.
 
-The cluster has several APIs and Interfaces. The Kubernetes API of the Control Plane is only available to Giant Swarm operations personnel and only through the Bastion hosts. 
+The cluster has several APIs and Interfaces. The Kubernetes API of the Control Plane is only available to Giant Swarm operations personnel and only through the Bastion hosts.
```
Comment (Oshratn) [inline src/content/basics/azure-architecture/index.md]: Is it "Bastions hosts", "Bastion hosts" ?
It's not consistent in the document.

File: src/content/basics/azure-architecture/index.md [hunk 3]
```diff
@@ -14,29 +14,29 @@ Giant Swarm's Architecture is split into two logical parts, one being the Contro
...
 
 The Giant Swarm API, Monitoring and Alerting frontends as well as our Web User Interface, are exposed through the Control Plane Ingress Controller, which sits behind a public Load Balancer.
 
 ## Giant Swarm tenant cluster
 
-![Tenant Cluster Architecture](/img/architecture-azure-tenant-cluster.png)
+![Tenant Cluster Architecture](architecture-azure-tenant-cluster.png)
 
-Via the Giant Swarm API, our [CLI](https://github.com/giantswarm/gsctl), or our web interface, you can start Tenant Clusters of different sizes. There's a selection of recommended Azure instance types, which can be adjusted if needed. 
+Via the Giant Swarm API, our [CLI](https://github.com/giantswarm/gsctl), or our web interface, you can start Tenant Clusters of different sizes. There's a selection of recommended Azure instance types, which can be adjusted if needed.
```
Comment (Oshratn) [inline src/content/basics/azure-architecture/index.md]: This is the AWS document. I guess the reference to Azure is a copy-paste error.

File: src/content/basics/azure-architecture/index.md [hunk 4]
```diff
@@ -14,29 +14,29 @@ Giant Swarm's Architecture is split into two logical parts, one being the Contro
...
 
-Each cluster resides in its own Virtual Network. All Azure machines are in their own private subnet ([Networking Deepdive](https://blog.giantswarm.io/deep-dive-into-kubernetes-networking-in-azure/)). There are two possible access routes into the cluster. 
+Each cluster resides in its own Virtual Network. All Azure machines are in their own private subnet ([Networking Deepdive](https://blog.giantswarm.io/deep-dive-into-kubernetes-networking-in-azure/)). There are two possible access routes into the cluster.
```
Comment (Oshratn) [inline src/content/basics/azure-architecture/index.md]: Multiple references to Azure here as well, including a link to a blog post.

File: src/content/basics/azure-architecture/index.md (hunk 2 above)
Comment (snizhana-dynnyk) [reply src/content/basics/azure-architecture/index.md]: changed to 'Bastion hosts' everywhere

File: src/content/basics/aws-architecture/index.md (hunk 1 above)
Comment (snizhana-dynnyk) [reply src/content/basics/aws-architecture/index.md]: changed everywhere to  "our Web User Interface"

**Comment by Oshratn** [comment]
LGTM

**Comment by marians** [comment]
Thanks @snizhana-dynnyk ! As a general rule of thumb, I recommend to not mix too many concerns in one PR, as that makes reviews more difficult.

**Pull request merged**